backends/__init__.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+__all__ = ['AbstractBackend', 'FileBackend']
+
+class AbstractBackend(object):
+
+    def __init__(self, *args, **kwargs):
+        super(AbstractBackend, self).__init__(*args, **kwargs)
+
+    def get_all_events(self):
+        """
+        Must return a dictionary with the form:
+
+        {
+            "<thread_name_1>": {
+                "name": "<event_name>",
+                "description": "<description_1>",
+                "yes": ["<email_1>", "<email_2>", ...],
+                "no": ["<email_1>", "<email_2>", ...],
+                "maybe": ["<email_1>", "<email_2>", ...],
+                "creator": int(<creator_user_id>),
+                "duration": int(<duration_seconds>) or None,
+                "place": "<place_name>" or None,
+                "time": "<%H:%m>" or None, # refer to http://strftime.org/ for % meaning.
+                "date": "<%Y-%m-%d>" or None,
+                "limit": int(<max_limit_people>) or None
+                "calendar_event": {
+                    "html_link": "<cal_event_url>",
+                    "id": "<cal_event_id>",
```

Comment:
- What is the cal_event_id used for? We won't have one for Google Calendar, but we will have our own.